scrapper.py

```diff
@@ -27,71 +53,164 @@ class UnknownConfigError(Exception):
     """
 
 
+lw = LinkWorker('', '')
```

Comment:
???

Reply:
we need it to get absolute link from relative
```suggestion
lw = LinkWorker('', '')
```